src/Server_TEST.cc

```diff
@@ -39,23 +39,12 @@
 
 #include "plugins/MockSystem.hh"
 #include "../test/helpers/Relay.hh"
+#include "../test/helpers/EnvTestFixture.hh"
 
 using namespace ignition;
 using namespace ignition::gazebo;
 using namespace std::chrono_literals;
 
-class ServerFixture : public ::testing::TestWithParam<int>
-{
-  protected: void SetUp() override
-  {
-    // Augment the system plugin path.  In SetUp to avoid test order issues.
-    ignition::common::setenv("IGN_GAZEBO_SYSTEM_PLUGIN_PATH",
-           (std::string(PROJECT_BINARY_PATH) + "/lib").c_str());
-
-    ignition::common::Console::SetVerbosity(4);
-  }
-};
```

Comment:
I believe this is breaking ABI :monocle_face: https://build.osrfoundation.org/job/ignition_gazebo-abichecker-any_to_any-ubuntu_auto-amd64/4020/API_5fABI_20report/

Reply:
I believe that build failed because of server connectivity issues, see the logs here:

https://build.osrfoundation.org/job/ignition_gazebo-abichecker-any_to_any-ubuntu_auto-amd64/4020/console

Note how the ABI report says "Compatibility 100%" and doesn't have any symbols marked with some severity. This means we're good :smile: 

It's surprising that the ABI checker is considering that the `TestFixture` symbols were added by this PR, when they have been added by #926 :woman_shrugging: 

Also, note that this PR doesn't touch any headers that are installed, so there's no possibility of breaking ABI.